OpenRA.Game/Network/UnitOrders.cs

```diff
@@ -18,7 +18,9 @@ namespace OpenRA.Network
 {
 	static class UnitOrders
 	{
-		static Player FindPlayerByClient(this World world, Session.Client c)
+    private const string BattlefieldControlChatName = "Battlefield Control";
```

Comment:
In my opinion `BattlefieldControlChatName` is too long, you could use something short like `AnnouncerName`.